composer.json

```
@@ -2,8 +2,10 @@
   "name": "pimcore/output-data-config-toolkit-bundle",
   "license": "GPL-3.0+",
   "type": "pimcore-bundle",
+  "prefer-stable": true,
+  "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^10.5 || ^11@dev",
+    "pimcore/pimcore": "^10.6 || ^11@dev",
```

```suggestion
    "pimcore/pimcore": "^10.6 || ^11.0",
```
since minimum stability now set to dev